docs/production/deployment.md

```diff
@@ -211,78 +198,20 @@ directory if you want to customize. A good example of doing this is
-[full-text search](../subsystems/full-text-search.md) experience
-around issues dictionary files are relevant (e.g. stemming).
-
-You also need to pass some extra options to the Zulip installer in
-order to avoid it throwing an error when Zulip attempts to configure
-the database's dictionary files for full-text search; the details are
-below.
-
-#### Step 1: Set up Zulip
-
-Follow the [standard instructions](install.md), with modified `install`
-arguments:
+For example, to install a Zulip Redis server on a machine, you can run
+the following after unpacking a Zulip production release tarball:
 
 ```bash
-./zulip-server-*/scripts/setup/install --certbot \
-    --email=YOUR_EMAIL --hostname=YOUR_HOSTNAME \
-    --puppet-classes=zulip::profile::standalone_nodb \
-    --postgresql-missing-dictionaries
-```
-
-#### Step 2: Create the PostgreSQL database
-
-Access an administrative `psql` shell on your PostgreSQL database, and
-run the commands in `scripts/setup/create-db.sql` to:
-
-- Create a database called `zulip`.
-- Create a user called `zulip`.
-- Now log in with the `zulip` user to create a schema called
-  `zulip` in the `zulip` database. You might have to grant `create`
-  privileges first for the `zulip` user to do this.
-
-Depending on how authentication works for your PostgreSQL installation,
-you may also need to set a password for the Zulip user, generate a
-client certificate, or similar; consult the documentation for your
-database provider for the available options.
-
-#### Step 3: Configure Zulip to use the PostgreSQL database
-
-In `/etc/zulip/settings.py` on your Zulip server, configure the
-following settings with details for how to connect to your PostgreSQL
-server. Your database provider should provide these details.
-
-- `REMOTE_POSTGRES_HOST`: Name or IP address of the PostgreSQL server.
-- `REMOTE_POSTGRES_PORT`: Port on the PostgreSQL server.
-- `REMOTE_POSTGRES_SSLMODE`: SSL Mode used to connect to the server.
-
-If you're using password authentication, you should specify the
-password of the `zulip` user in /etc/zulip/zulip-secrets.conf as
-follows:
-
-```ini
-postgres_password = abcd1234
+./scripts/setup/install --puppet-classes zulip::profile::redis
 ```
 
-Now complete the installation by running the following commands.
+To run the database on a separate server, including a cloud provider's managed
+PostgreSQL instance (e.g. AWS RDS), see out [dedicated PostgreSQL
+documentation][postgresql].
 
-```bash
-# Ask Zulip installer to initialize the PostgreSQL database.
-su zulip -c '/home/zulip/deployments/current/scripts/setup/initialize-database'
-
-# And then generate a realm creation link:
-su zulip -c '/home/zulip/deployments/current/manage.py generate_realm_creation_link'
-```
+[standalone.pp]: https://github.com/zulip/zulip/blob/main/puppet/zulip/manifests/profile/standalone.pp
+[zulipchat-puppet]: https://github.com/zulip/zulip/tree/8.x/puppet/zulip_ops/manifests
```

Comment:
This line was fudged a bit, since `tree/main/puppet/zulip_ops` doesn't exist, and it complains about `tree/main/puppet/kandra` since that doesn't exist in this branch.

Reply:
Seems like the right solution to me!